analyzers/its/expected/akka.net/Akka.DistributedData--netstandard2.0-S1944.json

```diff
@@ -0,0 +1,43 @@
+{
+"issues":  [
+{
+"id":  "S1944",
+"message":  "Review this cast; in this project there's no type that implements both 'ORDictionary<TKey, LWWRegister<TValue>>.IDeltaOperation' and 'ORDictionary.IDeltaOperation'.",
+"location":  {
+"uri":  "sources\akka.net\src\contrib\cluster\Akka.DistributedData\LWWDictionary.cs",
+"region":  {
+"startLine":  404,
```

Comment:
These looks like TP to me. `ORDictionary<TKey, LWWRegister<TValue>>.IDeltaOperation` doesn't seem to inherit `ORDictionary.IDeltaOperation`. Am I missing something?